setup.py

```diff
@@ -942,17 +943,17 @@ def __init__(self, attrs=None):
         with open(SALT_LONG_DESCRIPTION_FILE, encoding="utf-8") as f:
             self.long_description = f.read()
         self.long_description_content_type = "text/x-rst"
-        self.python_requires = ">=3.5"
+        self.python_requires = ">=3.6"
```

Comment:
```suggestion
        self.python_requires = ">=3.7"
```

Reply:
I did have ">=3.7" but tests failed on Py 3.6, which prompted that question https://saltstack.slack.com/archives/C02D6MGTGF3/p1671162285865489

Hence the reversion to ">=3.6" so that tests would pass.